docs/conf.py

```diff
@@ -71,6 +71,7 @@
     "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
     "sklearn": ("http://scikit-learn.org/dev", None),
     "hyppo": ("https://hyppo.neurodata.io", None),
+    "matplotlib": ("http://matplotlib.org/3.1.3/tutorials/colors/%s", "matplotlib")
```

Comment:
```suggestion
    "matplotlib": ('https://matplotlib.org', None)
```

Reply:
see https://stackoverflow.com/questions/21538983/specifying-targets-for-intersphinx-links-to-numpy-scipy-and-matplotlib?noredirect=1&lq=1